### ISSUE TITLE
[Bug] Wyze Lock status isn't updated on manual locking/unlocking

### ISSUE THREAD

**Issue by aleck011mi** [JoshuaMulliken/ha-wyzeapi#351]
**Describe the bug**
Since a couple versions ago, my Wyze Lock's status (open/close, locked/unlocked) is no longer reported to HA when changed physically via the lock.

**To Reproduce**

1. Properly setup Wyze Lock via HACS integration (been setup for over 6 months in my case)
2. Lock or unlock lock via HA, note status updates properly in HA
3. Lock/unlock and open/close door physically. Wait a few seconds between each action. Note no change in status in HA.

**Expected behavior**
Within a couple seconds of altering the status of the lock and door physically, I expect HA to have the correct status.

**System configuration**
System: Docker (Synology 918+)
HA Version: 2022.2 (latest)
WyzeApi Version: Latest version included in this Integration.

**home-assistant.log**

```
<PUT YOUR LOG HERE>
```

I do not have this setup yet, but would be happy to do it if having problems reproducing.

Alec

**Comment by JoshuaMulliken** [comment]
Yes this is because of the rate limits on our ability to pull data from Wyze API.

I doubt this will get fixed.

Thanks,

Josh

**Comment by yoinx** [comment]
To add to this. The lock is polled about one every 15 seconds (which may be adjusted slower based on the number of devices being polled). 

As Josh said, the API limitations imposed by Wyze drastically slowed down how fast this integration can detect changes.

**Comment by bgou** [comment]
Can we modify the pull frequency to 1 minute or allow users to set it themselves?

**Comment by yoinx** [comment]
> Can we modify the pull frequency to 1 minute or allow users to set it themselves?

I mean, setting it to once a minute isn't going to do much but make it 4x slower...

Anyone can edit the code on their instance to make things whatever they want.

**Comment by aleck011mi** [comment]
I'm resetting up my lock as you all are implying that the lock status should still be updating. Currently, I am not getting any updates at all. 15 seconds, even a minute is sufficient for my uses. Thanks as always to all the work on this project.

**Comment by LordNex** [comment]
I’ve been describing this same issue since implementation. It was blown off as connectivity issues and dropped

LoRdNeX

On Feb 21, 2022, at 12:46 PM, Joe Schubert ***@***.***> wrote:

﻿

To add to this. The lock is polled about one every 15 seconds (which may be adjusted slower based on the number of devices being polled).

As Josh said, the API limitations imposed by Wyze drastically slowed down how fast this integration can detect changes.

**Comment by aleck011mi** [comment]
In removing my lock from the Wyze app and pairing it from scratch, both lock/unlock and door open/close is now working again. Door open/close has more latency than lock/unlock but all is working for my purposes. Thanks all, I'm closing this as I don't believe this was an issue with code on this Integration. Appreciate everyone's thoughts on this.
